css/styles.css

```diff
@@ -319,3 +312,45 @@ button:focus {
   outline: none;
 }
 
+
+@media screen and (width <= 768px) {
```

Comment:
Here we start the style overrides for small screens. The Chrome (and I think Firefox too) has handy suggestions for common screen size breakpoints (though they're just suggestions):

![image](https://github.com/user-attachments/assets/06c57128-1ffa-4349-aeda-bcf4d7356a94)